EdvClientCache.js

```diff
@@ -3,9 +3,11 @@
  */
 'use strict';
 
+import cache from './cache';
+
 export default class EdvClientCache {
   constructor() {
-    this.cache = new Map();
+    this.cache = cache.set('edv-client-cache', new Map());
```

Comment:
Why not make the edv client cache itself also an LRU cache? Do we even need an edv client cache anymore? What does it buy us?

Reply:
Good catch. It should be an LRU cache. Do we need it and what does it buy us? I would need to look into where it's being used and for what reasons. The removal of it should be filed in an issue and handled in a separate PR. I will update it to use an LRU cache instead of  a Map in this PR.

Reply:
Actually, I don't think this even works -- what happens when `cache` from `./cache.js` wipes out `edv-client-cache`? This will break. That top level cache as an LRU cache doesn't make sense.

Reply:
Though there is a possibility that one can setup an LRU cache with `maxItems` and `maxAge` we do not for the top level cache. To be on the safe side, we can move to use a Map as that is how the top level cache functions.

Reply:
Yeah, an LRU cache isn't the right primitive for the top level "cache". It's not even a cache itself. It's a container for caches.

Reply:
Correct. Submitted a fix.